test/services/bill-runs/two-part-tariff/submit-amended-billable-returns.service.test.js

```diff
@@ -0,0 +1,69 @@
+'use strict'
+
+// Test framework dependencies
+const Lab = require('@hapi/lab')
+const Code = require('@hapi/code')
+
+const { describe, it, beforeEach } = exports.lab = Lab.script()
+const { expect } = Code
+
+// Test helpers
+const DatabaseSupport = require('../../../support/database.js')
+const ReviewChargeElementHelper = require('../../../support/helpers/review-charge-element.helper.js')
+const ReviewChargeElementModel = require('../../../../app/models/review-charge-element.model.js')
+
+// Thing under test
+const SubmitAmendedBillableReturnsService = require('../../../../app/services/bill-runs/two-part-tariff/submit-amended-billable-returns.service.js')
+
+describe('Submit Amended Billable Returns Service', () => {
```

Comment:
Need to add a test to check when the validation fails and there is a value in `validationResult`